src/Features/CSharp/Portable/Completion/KeywordRecommenders/IntoKeywordRecommender.cs

```diff
@@ -40,6 +40,17 @@ private bool IsValidContextForSelect(CSharpSyntaxContext context)
 
             var lastToken = select.Expression.GetLastToken(includeSkipped: true);
 
+            var memberAccessExpression = select.Expression as MemberAccessExpressionSyntax;
+            if(memberAccessExpression  != null && string.IsNullOrEmpty(memberAccessExpression.Name.ToString()))
+            {
+                return false;
+            }
+
+            //if(lastToken.Kind() == SyntaxKind.DotToken)
```

Comment:
It did work, but I thought it was too simple. Would `token.Kind() == SyntaxKind.DotToken `be ok? When I look at other recommenders, that seems to be a common pattern. If I use `token.GetPreviousToken` i will get the token before the dot i.e an identifier?